vagrant/boxes.d/00-ubuntu.yaml

```diff
@@ -17,3 +17,20 @@ boxes:
     ansible:
       variables:
         ansible_python_interpreter: /usr/bin/python3
+  ubuntu2004:
+    box_name: 'generic/ubuntu2004'
+    pty: true
+    scenarios:
+      - foreman
+    shell: |
+      # Work around external nameservers in generic/ubuntu2004
```

Comment:
If only Ubuntu built libvirt boxes, then we wouldn't need this workaround :(

Reply:
exactly my thoughts earlier today.